internal/shard/shard_thread.go

```diff
@@ -110,7 +110,12 @@ func (shard *ShardThread) executeCommand(op *ops.StoreOp) []byte {
 		return diceerrors.NewErrWithFormattedMessage("unknown command '%s', with args beginning with: %s", op.Cmd.Cmd, strings.Join(op.Cmd.Args, " "))
 	}
 
-	// The following commands could be handled at the server level, however, we can randomly let any shard handle them
+	// Till the time we refactor to handle QWATCH differently using HTTP Streaming/SSE
+	if op.HTTPOp {
+		return diceCmd.Eval(op.Cmd.Args, shard.store)
+	}
```

Comment:
Won't this cause a segfault in case the user ends up triggering QWATCH or QUNWATCH?

Reply:
Yes, there is a separate issue to handle QWATCH etc with HTTP. Once that is added, this can be removed.

Reply:
This will cause the server to crash, instead let's return an UnimplementedError for such commands.

Reply:
Okay cool.